sources/manticore-worker/Dockerfile

```diff
@@ -1,20 +1,23 @@
-FROM manticoresearch/helm-balancer:5.0.0.0 AS builder
-
 FROM manticoresearch/manticore:5.0.0
 
 RUN apt --fix-broken install -y && echo 'debconf debconf/frontend select Noninteractive' | debconf-set-selections && \
-apt-get update && apt-get install -y curl php-cli php-mysqlnd php-curl wget && rm -rf /var/lib/apt/lists/*
-
-COPY --from=builder /var/www/localhost/ /etc/manticoresearch/
+apt-get update && apt-get update && apt -y install software-properties-common && add-apt-repository -y ppa:ondrej/php && apt-get update && \
```

Comment:
- Remove extra `apt-get update`
- Nit: Better to add necessary repos and then do a single `apt-get update`